zrtech_v2/pinmap.tcl

```diff
@@ -0,0 +1,11 @@
+#
+# Clock
+#
+set_location_assignment PIN_24 -to clk
+set_instance_assignment -name IO_STANDARD "3.3-V LVTTL" -to clk
+
+#
+# LED D2 (DS_DP)
+#
+set_location_assignment PIN_3 -to q
+set_instance_assignment -name IO_STANDARD "3.3-V LVTTL" -to clk
```

Comment:
Should this be `-to q` instead? I can amend locally if you want to avoid spinning a new patch

Reply:
indeed...
Fixed